README.md

```diff
@@ -0,0 +1,51 @@
+[![CharmHub Badge](https://charmhub.io/github-runner-operator/badge.svg)](https://charmhub.io/github-runner-operator)
+[![Promote charm](https://github.com/canonical/github-runner-operator/actions/workflows/promote_charm.yaml/badge.svg)](https://github.com/canonical/github-runner-operator/actions/workflows/promote_charm.yaml)
+[![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)
+
+# GitHub runner
+
+## Description
+
+This machine charm creates self-hosted GitHub runners. Each unit of this charm will start a configurable number of LXD based containers and virtual
+machines to host them. Each runner performs only one job, after which it unregisters from GitHub to ensure that each job runs in
+a clean environment.
+
+The charm will periodically check the number of idle runners and spawn or destroy runners as necessary to match the number provided by configuration of
```

Comment:
The documentation is wrong.
Currently, it is the total number of runners. It can be changed to number of idle runners, but a consistent number of runner with consistent resource usage is likely a better design.
The documentation should be updated.

Reply:
Minor comments